ktor-client/ktor-client-features/ktor-client-json/common/src/io/ktor/client/features/json/JsonFeature.kt

```diff
@@ -66,10 +66,20 @@ class JsonFeature internal constructor(
          */
         @KtorExperimentalAPI
         var acceptContentTypes: List<ContentType> = listOf(ContentType.Application.Json)
+            @Deprecated("Use accept() function instead.")
```

Comment:
ReplaceWith

Reply:
How can we provide it? This is not possible, I believe.

Reply:
I added `replaceWith = ReplaceWith("accept(newList, onlyThese = true)")`. Not sure if it works, but hope so.

Reply:
No, it doesn't (just tested). And it couldn't. Because setter is used by all different operators like `=`, `+=`, `-=` and so on.

Reply:
Note that deprecation itself works well for all the operators, but not replacements.

Reply:
Thanks for testing, but I don't see any reason why it could not work in theory. The Idea knows how to replace `+=` and `-=` to `list = list + ...` and at the same time it's clear how to make the replacement of `= list` to `accept(list)`. Hope it will be implemented eventually :)

Reply:
I see, IDEA could do it, right. The only issue is that `accept` is not `=`, it's `+=`. So there is no valid replacement for `-=` :)

Reply:
Well, that's the truth. So, what's the decision? Do you think it makes sense to remove the replacement?